sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -136,6 +141,7 @@ public partial class ClientPipelineOptions
     {
         public ClientPipelineOptions() { }
         public System.ClientModel.Primitives.ClientLoggingOptions? ClientLoggingOptions { get { throw null; } set { } }
+        public bool? EnableDistributedTracing { get { throw null; } set { } }
```

Comment:
Curious why this is nullable - what is the default behavior?

Reply:
For this one I just followed the existing pattern since the other properties are set to null by default. The default value of this property is null, but if it is null SCM treats it as true.

Reply:
Yeah I see the other bool properties are similarly nullable.